USFMToolsSharp/Models/Markers/UnknownEndMarker.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace USFMToolsSharp.Models.Markers
+{
+    public class UnknownEndMarker : Marker
```

Comment:
I don't know if this is needed, since unknown marker would handle this just fine. I'm ok with adding it though. I'm curious as to what it is needed for.

Reply:
See comment above